clients/amqp-connection/src/main/java/org/eclipse/hono/client/amqp/tracing/MessageAnnotationsExtractAdapter.java

```diff
@@ -1,5 +1,5 @@
 /*******************************************************************************
- * Copyright (c) 2016, 2022 Contributors to the Eclipse Foundation
+ * Copyright (c) 2016, 2023 Contributors to the Eclipse Foundation
```

Comment:
Just for your information, there is no formal requirement to maintain and adjust the period over which changes have occurred in the header (anymore). After a talk I had with Wayne Beaton at EclipseCon, I have adopted a habit of removing the upper boundary whenever I touch a file these days.